app/services/bill-runs/two-part-tariff/submit-remove-bill-run-licence.service.js

```diff
@@ -53,15 +56,27 @@ async function _allLicencesRemoved (billRunId) {
   return false
 }
 
-async function _flagForSupplementaryBilling (licenceId) {
+async function _fetchLicenceRef (licenceId) {
   const licence = await LicenceModel.query()
     .findById(licenceId)
-    .patch({ includeInSrocTptBilling: true })
     .returning('licenceRef')
 
   return licence.licenceRef
 }
 
+async function _flagForSupplementaryBilling (licenceId, billRunId) {
+  const financialYearEnd = await BillRunModel.query()
+    .findById(billRunId)
+    .returning('toFinancialYearEnding')
+
+  await LicenceSupplementaryYearModel.query()
```

Comment:
As this is awaited above and there are no statements that follow it in the function, we should instead

```suggestion
  return LicenceSupplementaryYearModel.query()
```